content/_data/logo/oslo.yml

```diff
@@ -0,0 +1,8 @@
+---
+name: 'Oslo'
+url: '/images/logos/oslo/oslo.png'
+url_256: '/images/logos/oslo/oslo256.png'
```

Comment:
Not sure it is official logo. No prob with publishing, but it needs polishing IMHO

Reply:
Should I remove this for now?

Reply:
have removed it for now.